pmc/operations/default.json

```diff
@@ -54,7 +54,7 @@
           "articles_over_time": {
             "date_histogram": {
               "field": "timestamp",
-              "interval": "month"
+              "fixed_interval": "month"
```

Comment:
This fails with a rather cryptic error message:

```json
{
  "error" : {
    "root_cause" : [
      {
        "type" : "x_content_parse_exception",
        "reason" : "[8:33] [date_histogram] failed to parse field [fixed_interval]"
      }
    ],
    "type" : "x_content_parse_exception",
    "reason" : "[8:33] [date_histogram] failed to parse field [fixed_interval]",
    "caused_by" : {
      "type" : "illegal_argument_exception",
      "reason" : "failed to parse [month]",
      "caused_by" : {
        "type" : "number_format_exception",
        "reason" : "For input string: \"mont\""
      }
    }
  },
  "status" : 400
}
```

digging in the [docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-datehistogram-aggregation.html#_fixed_intervals) reveals the cause:

> [...] fixed intervals cannot express other units such as months, since the duration of a month is not a fixed quantity. Attempting to specify a calendar interval like month or quarter will throw an exception.
>
> If we try to recreate the "month" calendar_interval from earlier, we can approximate that with 30 fixed days

Can you please adjust this accordingly (here and in the other affected places)? I'd expect that this *might* also change latency numbers. As a follow-up to this PR, can you please check our nightlies and add an according annotation should that be the case?

Reply:
Upon reconsideration I'd suggest that we move to `calendar_interval` here for the simple reason that it also broadens the coverage of our benchmarks, i.e. with benchmarks for `fixed_interval` and `calendar_interval` we'd notice regressions over time in both.

Reply:
👍